forem/forem
# Member index: add copy email to clipboard functionality

## What type of PR is this? (check all applicable)


## Description

This PR adds the overflow actions menu to each user entry on the new member index page. The menu contains only one action - 'copy email' which should... you guessed it... copy the user's email to the clipboard 😄 

## Related Tickets & Documents



- Related Issue https://github.com/forem/forem/issues/16911

## QA Instructions, Screenshots, Recordings

- Make sure `member_index_view` feature flag is enabled, then visit `/admin/users`
- For any entry in the table, click the overflow menu and click the "copy to clipboard" button
- Check the snackbar confirmation appears
- Check that the correct email address was copied to your clipboard (paste it somewhere!)
- Make sure the overflow menu works in the large table layout, and the smaller 'mobile friendly' layout

### UI accessibility concerns?

For performance, we're not using the `initializeDropdown` function, but I've made sure to follow all the accessibility best practices baked into that helper, so it works by keyboard / has appropriate labels etc.

## Added/updated tests?



## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [X] This change does not need to be communicated, and this is why not: Part of a larger issue

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![copy that](https://media.giphy.com/media/1AjCJ5gePMQhgZ1U4A/giphy.gif)
